doc/blameable.md

```diff
@@ -4,10 +4,19 @@ The **Blameable** behavior automates the update of user information on your Doct
 
 ## Index
 
-- [Getting Started](#getting-started)
-- [Configuring Blameable Objects](#configuring-blameable-objects)
-- [Using Traits](#using-traits)
-- [Logging Changes For Specific Actions](#logging-changes-for-specific-actions)
+- [Blameable Behavior Extension for Doctrine](#blameable-behavior-extension-for-doctrine)
```

Comment:
Just a personal suggestion, but I'd leave the index as it was where only the top level headers are listed.  Since this project isn't using an external docs tool and they're being rendered in GitHub, GitHub's native outline feature keeps this a little more maintainable (GitHub will automatically outline/index the whole document, all we have to do is link to the main sections).

Reply:
Reverted the TOC back to a simplified version